e2e/test_get_code.py

```diff
@@ -1,14 +1,18 @@
 import os
 import unittest
 
-from src.fetch.contracts import ContractDetector
+from src.fetch.contracts import EvmAccountInfo
+from src.files import NetworkFile
 
 NODE_URL = os.environ['NODE_URL']
 
+test_file = NetworkFile(name="test-file.csv", path='./out/test')
```

Comment:
Why `NetworkFile` when it has nothing to do with networks?
Why is it only now needed?

Reply:
This class is compatible with both networks (mainnet and xdai). I guess in this case the file itself is not network specific I could probably change this. We use network file elsewhere in the project for the purpose of appending the network name to the file name on write.